blog/_posts/btcpay-server-1-4-0.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "BTCPay Server 1.4.0 - The Year of Change"
+date: 2022-01-27
+author: Zaxounette
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "ui-ux"
+  - "greenfield"
+  - "api"
+  - "update"
+coverImage: "/images/BTCPayServer1.4.0.png"
+---
+
+At BTCPay Server, the year 2022 starts out with a bang!
+We are today releasing **BTCPay Server version 1.4.0**. While this release has no code breaking changes, this update contains huge UI changes that may take some time to get used to, particularly for our power users and compagnies that may have high-volume instances running that requires staff to access their BTCPay Server. **We recommend those community members to spin up a fresh instance**, or to check out our [testing instance](https://chat.btcpayserver.org/btcpayserver/channels/testers) to get a feel of the massive UI changes that are included before updating their production instance.
+
+**BTCPay Server 1.4.0** is a major update that includes multiple items ranging from our massive UI revamp to updates to our Greenfield API. We won't cover them all in this release post, but as usual, the high-time preference crowd can view the [full Changelog](https://github.com/btcpayserver/btcpayserver/releases) directly.
+
+To update, simply navigate to your **Server Settings > Maintenance** tab and click **Update** or type *btcpay-update.sh* in the command line.
+
+## **UI overhaul** 🎨
+
+After six months of scoping, iterating and nearly two months of implementation tweaking and testing, we are finally ready to move to the next chapter of the BTCPay Server book.
+The goal behind these chages was to make the initial configuration as well as the first store creation and wallet setup as easy as possible for users who are new to BTCPay Server, while still enabling our power users to have easy access to the more complexe features available in BTCPay Server.
+
+![BTCPay 1.4.0 Payment Methods view](/images/1.4.0_payment_methods.png)
+
+Without going into too much details (that you can find in our fully transparent [roadmap](https://github.com/orgs/btcpayserver/projects/19)), this overhaul is but a step into getting plugins as well as centralized store and instance dashboards into BTCPay Server.
+We hope you like these chages, and if you have any feedback feel free to reach out in our [community chat](https://chat.btcpayserver.org/), we'll be glad to have you.
+
```

Comment:
@Zaxounette  We really should let people know about our new invoice statuses change (we have a warning on the invoice page)
- Paid = Settled 
- Processing = Settled 

We should highlight new features here:
- Login codes
- Two factor authentication with your Lightning wallet (LNURL-Auth)

Reply:
Also we should talk more about plugins afterwards, afterall we are releasing LNBank, we have to mention it.

Reply:
Regarding Plugins, noted.

Regarding invoice status change, I mention it in "Misc".

Reply:
Imo invoice statuses should not be in misc, it's a big change and we should emphasize it as people's processes depend on it.